packages/react-components/priority-overflow/src/overflowManager.ts

```diff
@@ -24,7 +25,9 @@ export function createOverflowManager(): OverflowManager {
   };
 
   const overflowItems: Record<string, OverflowItemEntry> = {};
-  const overflowGroups: Record<string, { visibleItemIds: Set<string>; invisibleItemIds: Set<string> }> = {};
+  const overflowDividers: Record<string, OverflowDivider> = {};
+  const groups: Record<string, { visibleItemIds: Set<string>; invisibleItemIds: Set<string> }> = {};
```

Comment:
let's abstract all group related functionalities completely, like we did when we paired offline

```ts
const groupManager = createGroupManager();

groupManager.addItem();
```

Reply:
fixed